mods/creative/init.lua

```diff
@@ -54,7 +57,7 @@ function minetest.handle_node_drops(pos, drops, digger)
 	end
 	local inv = digger:get_inventory()
 	if inv then
-		for _, item in ipairs(drops) do
+		for _, item in pairs(drops) do
```

Comment:
Indeed, with numbers (like above) ipairs is faster, but with functions and strings it seems pairs is better.